content/plugins/lara-zeus-matrix-choice.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Matrix Choice ⚡️ Multiple Choice Grid
+slug: lara-zeus-matrix-choice
+author_slug: lara-zeus
+categories: [form-builder,form-field]
+description: multiple choice grid form component.
+discord_url: 
+docs_url: https://raw.githubusercontent.com/lara-zeus/matrix-choice/3.x/docs/filament.md
+url: https://larazeus.com/matrix-choice
+github_repository: lara-zeus/matrix-choice
+has_dark_theme: true
+has_translations: true
+thumbnail: lara-zeus-matrix-choice.png
```

Comment:
Please remove the 'thumbnail' key. It will automatically be generated.

Reply:
remove it from other packages too 👍🏽